File: wrapupsnacks/python/lowest_common_multiple.py


first_number = int(input("Enter first number: "))
second_number = int(input("Enter second number: "))


highestNumber=first_number
if second_number>first_number:

	highestNumber = second_number

lowestCommonMultiple = highestNumber*2

while lowestCommonMultiple%first_number!=0 or lowestCommonMultiple % second_number!=0:
	lowestCommonMultiple+=1

	
print(lowestCommonMultiple)
	

File: wrapupsnacks/java/LowestCommonMultiple.java

import java.util.Scanner;
public class LowestCommonMultiple{
public static void main(String [] args){

Scanner scanner = new Scanner(System.in);

System.out.print("Enter first number: ");

int firstNumber = scanner.nextInt();

System.out.print("Enter second number: ");
int secondNumber = scanner.nextInt();


int highestNumber=firstNumber;
if(secondNumber>firstNumber) highestNumber = secondNumber;

int lowestCommonMultiple= highestNumber*2;

while(lowestCommonMultiple%firstNumber!=0 || lowestCommonMultiple % secondNumber!=0){
	lowestCommonMultiple++;

	}
	System.out.println(lowestCommonMultiple);
	
}

}


